rift/interface.py

```diff
@@ -1404,7 +1404,12 @@ def process_rx_tire_packet(self, tire_packet):
     def neighbor_direction(self):
         if self.neighbor is None:
             return None
+
+        # Cannot determine current node level, we can't infer the neighbor direction
```

Comment:
@brunorijsman We discovered that this was causing a crash when a fallen leaf situation is reached.
Disconnecting a spine (with level = undefined) from all the ToFs of its plane, it was not able to determine its level